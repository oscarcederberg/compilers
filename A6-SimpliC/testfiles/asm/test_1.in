int main() {
  int x = 121;
  print(1);
  int y = 12321;
  int z = 32123;
  print(y);
  print(z);
  print(x);
  return 0;
}