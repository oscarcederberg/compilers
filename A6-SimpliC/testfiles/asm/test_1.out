1
12321
12321
121
