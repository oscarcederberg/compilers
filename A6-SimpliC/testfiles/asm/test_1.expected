1
12321
32123
121